bokehjs/src/lib/models/widgets/tables/data_table.ts

```diff
@@ -97,8 +130,8 @@ export class DataProvider {
 export class DataTableView extends WidgetView {
   model: DataTable
 
-  private data: DataProvider
-  private grid: SlickGrid
+  protected data: DataProvider
+  protected grid: SlickGrid
```

Comment:
This is a problem, because this exposes external types in the public API, making working with bokehjs a huge pain downstream. It's a possibility that each implementation using slick grid will have to keep its private data/grid and initialize those on its own.